src/ImageSharp/Formats/Png/PngDecoderCore.cs

```diff
@@ -112,6 +112,11 @@ internal sealed class PngDecoderCore : IImageDecoderInternals
         /// </summary>
         private PngChunk? nextChunk;
 
+        /// <summary>
+        /// "Exif" and two zero bytes. Used for the legacy exif parsing.
+        /// </summary>
+        private static readonly byte[] ExifHeader = new byte[] { 0x45, 0x78, 0x69, 0x66, 0x00, 0x00 };
```

Comment:
```suggestion
        // This uses C# compiler's optimization to refer to the static data directly, no intermediate array allocations happen.
        private static ReadOnlySpan<byte> ExifHeader => new byte[] { 0x45, 0x78, 0x69, 0x66, 0x00, 0x00 };
```
For background info on that see e.g. https://vcsjones.dev/csharp-readonly-span-bytes-static/